__tests__/seed.test.js

```diff
@@ -22,23 +22,45 @@ describe("/api/topics", () => {
         });
       });
   });
-test('GET 404: user enters invalid endpoint', () => {
+  test('GET 404: user enters invalid endpoint', () => {
     return request(app)
-    .get("/api/invalidURL")
-    .expect(404)
-    .then(({ body }) => {
+      .get("/api/invalidURL")
+      .expect(404)
+      .then(({ body }) => {
         const { message } = body
         expect(message).toBe('Not found')
-    })
+      })
   })
 })
 describe('/api', () => {
   test('GET 200: Returns all available endpoints', () => {
     return request(app)
-    .get('/api')
-    .expect(200)
-    .then(({ body }) => {
-      expect(body).toEqual(endpoints)
-    })
+      .get('/api')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(endpoints)
+      })
+  })
+})
+describe('/api/articles/:article_id', () => {
+  test('GET 200: responds with article that corresponds the article_id', () => {
+    return request(app)
+      .get('/api/articles/4')
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body
+        expect(article).toMatchObject(expect.objectContaining(
+          {
```

Comment:
Is there a way we could make sure this test doesn't break if the test data ever gets updated? At the moment the test would break if someone changes the title of this article for example, so we don't want to check the actual values - try checking the types here instead! :D